src/hooks/use-arrow-key-navigation.ts

```diff
@@ -33,6 +33,14 @@ export type UseArrowKeyNavigationOptions = {
    * CSS selector which specifies the elements that navigation moves between
    */
   selector?: string;
+
+  /**
+   * Tells if the container is visible, ensuring no elements on it are focused
+   * while it is not.
+   * It also lets consumers control when the side effects should be re-computed.
+   * Defaults to `true`.
+   */
+  containerVisible?: boolean;
```

Comment:
This description is a bit vague. I would suggest something like: "Indicates if the container element is currently visible. This information is used to focus the current element when the container transitions from hidden to visible."

Reply:
Writing a lot of words and not say anything is my specialty 😅.

Thanks for the suggestion. I quite like it.